ports/esp32s2/components/bootloader/subproject/main/bootloader_start.c

```diff
@@ -412,14 +523,24 @@ static void board_led_on(void)
 static void board_led_off(void)
 {
 #ifdef NEOPIXEL_PIN
+  delay_cycle( ns2cycle(1000*1000) ) ;
```

Comment:
is there any reason that you move the delay before the neopixel off ? would the pixel is off when the delay is as it is ?

Reply:
I'll wait until you have pushed whatever changes you have made.  You are correct in that it works either way and should be after.  This is a remnant from trying to eliminate what I thought was a problem. Thanks